config.toml

```diff
@@ -6,7 +6,7 @@ theme = "cfbs-theme"
 # Number of posts per page
 Paginate = 10
 enableRobotsTXT = true
-disqusShortname = 'cfbuild-test'
+disqusShortname = 'cfbuild'
```

Comment:
@aleksandrychev where does this show up?

Reply:
Disqus shortcode uses this var in code. and on the UI I can see it her:
![image](https://user-images.githubusercontent.com/16775360/136341619-73d6128d-2b8c-4f14-aaec-40b10c2fb762.png)

Reply:
updated , now it's `cfengine-build`